pkg/stores/partition/listprocessor/processor.go

```diff
@@ -360,3 +394,28 @@ func PaginateList(list []unstructured.Unstructured, p Pagination) ([]unstructure
 	}
 	return list[offset : offset+p.pageSize], pages
 }
+
+func FilterByProjectsAndNamespaces(list []unstructured.Unstructured, projectsOrNamespaces ProjectsOrNamespacesFilter, namespaceCache corecontrollers.NamespaceCache) []unstructured.Unstructured {
+	if len(projectsOrNamespaces.filter) == 0 {
+		return list
+	}
+	result := []unstructured.Unstructured{}
+	for _, obj := range list {
+		namespaceName := obj.GetNamespace()
```

Comment:
Nit:
Might be good after this line to do a:

```golang
if namespaceName == ""{
    continue
}
```

I'm slightly uncomfortable on relying on `namespaceCache.Get("")` returning nil.